src/ccontrol/UserQueryFactory.cc

```diff
@@ -234,7 +234,7 @@ UserQueryFactory::UserQueryFactory(qproc::DatabaseModels::Ptr const& dbModels, s
     _userQuerySharedResources->queryMetadata->cleanupQueriesAtStart(_userQuerySharedResources->czarId);
 
     // Add logging context with czar ID
-    qmeta::CzarId czarId = _userQuerySharedResources->czarId;
+    CzarId czarId = _userQuerySharedResources->czarId;
```

Comment:
This is where `auto` could be helpful as well.
The second comment - if the value of the variable is not meant to be changed later in the block then:
```
CzarId const czarId = ...
```